src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -64,6 +68,17 @@ public function execute(array $input, array $options = array())
             $input = array_merge(array('command' => $this->command->getName()), $input);
         }
 
+        if ($this->inputs) {
+            if (null === $this->command->getHelperSet()) {
+                $helper = new QuestionHelper();
+                $this->command->setHelperSet(new HelperSet(array($helper)));
+            } else {
+                $helper = $this->command->getHelper('question');
+            }
+
+            $helper->setInputStream($this->createInputStream());
+        }
```

Comment:
@fabpot Only the QuestionHelper and SymfonyQuestionHelper have an `inputStream`, so I don't see a real benefit to make this looking for another object that the `question` helper.

I made this covering the case of SymfonyStyle is used (SymfonyQuestionHelper) in #18902, as this one is not part of the command helpset, we don't have any way to make it generic. It's [planned for 4.0 to move the setInputStream](https://github.com/symfony/symfony/issues/10844) method in the InputInterface, but ATM the QuesitonHelper is the only available place (and [documented] as is(http://symfony.com/doc/current/components/console/helpers/questionhelper.html#testing-a-command-that-expects-input)).
What do you think?